include/soci/session.h

```diff
@@ -215,6 +218,7 @@ class SOCI_DECL session
     SOCI_NOT_COPYABLE(session)
 
     void reset_after_move();
+    schema_table_name * alloc_schema_table_name(std::string & tableName);
```

Comment:
It doesn't look like the table name is modified by this function, so I think this should be

```suggestion
    schema_table_name * alloc_schema_table_name(const std::string& tableName);
```